cypress-tests/cypress/e2e/PaymentUtils/Nmi.js

```diff
@@ -351,20 +327,12 @@ export const connectorDetails = {
     },
     PaymentMethodIdMandate3DSManualCapture: {
       Request: {
-        payment_method: "card",
         payment_method_data: {
           card: successfulThreeDSTestCardDetails,
         },
         mandate_data: null,
         authentication_type: "three_ds",
-        customer_acceptance: {
-          acceptance_type: "offline",
-          accepted_at: "1963-05-03T04:07:52.723Z",
-          online: {
-            ip_address: "125.0.0.1",
-            user_agent: "amet irure esse",
-          },
-        },
+        customer_acceptance: customerAcceptance,
```

Comment:
would be helpful if you could verify the same with other config files as well.

Reply:
will take this in a separate PR, Thank you